agent/api/statechange.go

```diff
@@ -101,12 +101,20 @@ type AttachmentStateChange struct {
 	Attachment *apieni.ENIAttachment
 }
 
+type ErrShouldNotSendEvent struct {
+	resourceId string
+}
+
+func (e ErrShouldNotSendEvent) Error() string {
+	return fmt.Sprintf("Should not send events for internal tasks or containers: %s", e.resourceId)
```

Comment:
Should we start the error description with a lowercase as is the convention in Go?

Reply:
Fixed. Thanks!